src/assets/locales/en/translation.json

```diff
@@ -636,5 +637,76 @@
   "earn_strategy": {
     "withdraw_rewards_in_wrapped": "Withdraw rewards in {{wrappedCurrencySymbol}}:",
     "update_position": "Update position"
+  },
+  "transaction": {
+    "confirm_transaction_wallet": "Confirm this transaction in your wallet",
+    "confirm_transaction": "Confirm transaction",
+    "transaction_processing": "Transaction processing",
+    "transaction_failed": "Transaction failed",
+    "transaction_successful": "Transaction successful",
+    "swapping_to": "Swapping {{fromAmount}} {{fromCurrency}} to {{toAmount}} {{toCurrency}}",
+    "swapped_to": "Swapped {{fromAmount}} {{fromCurrency}} to {{toAmount}} {{toCurrency}}",
+    "adding_liquidity_to_pool": "Adding liquidity to {{poolName}} Pool",
+    "added_liquidity_to_pool": "Added liquidity to {{poolName}} Pool",
+    "removing_liquidity_from_pool": "Removing liquidity from {{poolName}} Pool",
+    "removed_liquidity_from_pool": "Removed liquidity from {{poolName}} Pool",
+    "claiming_pool_rewards": "Claiming pools rewards",
+    "claimed_pool_rewards": "Claimed pools rewards",
+    "issuing_amount": "Issuing {{amount}} {{currency}}",
+    "issued_amount": "Issuing {{amount}} {{currency}}",
+    "redeeming_amount": "Redeeming {{amount}} {{currency}}",
+    "redeemed_amount": "Redeemed {{amount}} {{currency}}",
+    "burning_amount": "Burning {{amount}} {{currency}}",
+    "burned_amount": "Burned {{amount}} {{currency}}",
+    "retrying_redeem_id": "Retrying redeem {{resquestId}}",
```

Comment:
```suggestion
    "retrying_redeem_id": "Receiving {{amount}} {{currency}}  compensation. Retry with a different Vault",
```

Reply:
Note: retry just gives you the compensation but does not actually retry to execute the request. Naming is a bit unfortunate here.

Reply:
@danielsimao if this is too long, then just `Receiving {{amount}} {{currency}}` (and also on the ones below) works as well

Reply:
But note that this is a notification, so I think we should only say `Receiving {{amount}} {{currency}} compensation due to failed redeem`

Reply:
Adding user actions like `Retry with a different Vault` does fit in a notification.

Reply:
Agree with you.